config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root to: 'users#index' # this will map the "https://<your-domain>/" URL
+  
+  get 'users/:id', to: 'users#show', as: 'user'
+  get 'users/:user_id/posts', to: 'posts#index'
+  get 'users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
```

Comment:
- [ ] [OPTIONAL] You can use the following format to structure your routes which is cleaner.

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
@ShadyShawkat I did that, but now the root is displaying the rails welcome page. And localhost/users is displaying the root content.

Is it still okay to leave it like that? 

![image](https://github.com/cancelei/blog-in-rails/assets/96393104/4d8589e0-7b69-4892-9df9-75a9656a1f89)

Reply:
@cancelei You can keep the root line in place and change the others to looks something like this:

```ruby 
root to: 'users#index'

resources :users, only: [:index, :show] do
  resources :posts, only: [:index, :show]
end
```

Reply:
It worked like a glove, thank you for the feedback!